### PR TITLE
chore: add note to bug report template

### PR DESCRIPTION
**Pull request by gracelu0**
### Issue # (if applicable)

N/A

### Reason for this change

Remind users to submit security-related issues using the existing mechanism in GitHub.

### Description of changes

Add note to the top of the template

### Description of how you validated changes

Previewed the template in GitHub (can't find a way to actually view the updated template locally)

### Checklist
- [n/a] My code adheres to the [CONTRIBUTING GUIDE](https://github.com/aws/aws-cdk/blob/main/CONTRIBUTING.md) and [DESIGN GUIDELINES](https://github.com/aws/aws-cdk/blob/main/docs/DESIGN_GUIDELINES.md)

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.yml
Comment (shikha372) [inline .github/ISSUE_TEMPLATE/bug-report.yml:15]: Thank you @gracelu0 , not sure if this is the template we should use related to issues that might not be related to any CVE per say, but i'll let team also take a look to chime in. IMO, if its not related to a CVE maybe we have a new type of bug template with security checkbox or related attributes to be populated.

Comment (marekaiv) [reply .github/ISSUE_TEMPLATE/bug-report.yml:15]: This is the link a customer would get if they selected "Report a security vulnerability" on the `New Issue` page.

Comment (moelasmar) [reply .github/ISSUE_TEMPLATE/bug-report.yml:15]: We will review the content of the `Report a security vulnerability` link with security team to review the best way how customers should report security issues. For now, we will refer the customers to this link, till we know what should be the content of that link.

**Review by moelasmar — APPROVED** [review]

**Comment by GavinZZ** [comment]
Comments on closed issues and PRs are hard for our team to see. If you need help, please open a new issue that references this one. If you wish to keep having a conversation with other community members under this issue feel free to do so.

**Pull request merged**
